src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../redux/home/home';
+import Card from './Card';
+
+export default function Home() {
```

Comment:
- [ ] As required by the previous reviewer and According to the project requirements, there should be a filtering mechanism on your Homepage (like a search functionality, for example). Kindly add this.

![sunny-home](https://user-images.githubusercontent.com/36542986/142707431-c8e72c2b-4016-4db5-abd9-2a2ae9a97405.jpg)